### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by LordMyschkin**
First try for a Pull Request to add german language to owntone und to finally enable language suppot in i18n.js

### CODE REVIEW

File: web-src/src/locales/de.json [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+        "add": "Hinzufügen",
+        "cancel": "Abbrechen",
+        "help": "Hinzufügen eines Podcasts führt zur Anlage einer RSS-Playlist, so kann Owntone das Podcast-Abo verwalten.",
+        "placeholder": "https://url-to-rss",
+        "processing": "Verarbeite…",
+        "title": "Füge Podcast RSS-Feed-Url hinzu"
+      },
+      "stream": {
+        "add": "Hinzufügen",
+        "cancel": "Cancel",
```
Comment (ejurgensen) [inline web-src/src/locales/de.json:23]: Not "Abbrechen"?

**Review by ejurgensen — COMMENTED** [review]
Wunderschön, vielen Dank :-)

I noticed a translation of "Cancel" that perhaps was missed, or maybe it was on purpose?

File: web-src/src/locales/de.json (hunk 1 above)
Comment (LordMyschkin) [reply web-src/src/locales/de.json:23]: oh, indeed. Should be "Abbrechen"

Comment (LordMyschkin) [reply web-src/src/locales/de.json:23]: and when scrolling through, I find some other mistakes as well - gonna correct them instantly.

**Comment by LordMyschkin** [comment]
I hope I did not miss anything else!
"Cancel" and "Albums" corrected now...

**Comment by ejurgensen** [comment]
Merged, thanks!

**Pull request merged**